packages/ninjam-server/PKGBUILD

```diff
@@ -5,37 +5,44 @@
 _name=ninjam
 pkgname=ninjam-server
 pkgver=0.080
-pkgrel=1
+pkgrel=2
+# git blame -l ninjam/server/ninjamsrv.cpp | grep "#define VERSION" | cut -d ' ' -f 1
+_commit=44c21b98042afc440af456e110f857b7947ff859
 pkgdesc='Ninjam online jamming server'
 arch=(x86_64 aarch64)
 url='https://cockos.com/ninjam/'
-license=(GPL2)
+license=(GPL-2.0-or-later)
 depends=(gcc-libs)
+makedepends=(git)
 groups=(pro-audio)
-conflicts=(ninjam-server-git)
-source=("$pkgname-$pkgver.tar.gz::https://github.com/cbix/$_name/archive/refs/tags/ninjamsrv/v$pkgver.tar.gz"
+source=("git+https://www-dev.cockos.com/ninjam/ninjam.git#commit=$_commit"
         "$_name.service"
         "$_name.sysusers")
-sha256sums=('64a5e2745ed335cc3e750dafde4f5b80405762cbda4def534486ea081adf3eac'
+sha256sums=('4f78057d1f49eb192daed33d2a9c856bf1c56326b2716d35656955bb99450470'
```

Comment:
Using a checksum with a Git source requires pacman/makepkg >= 6.1, which currently Manjaro does not have yet.

Since we sync this package to the AUR, maybe we should wait with this change, until pacman 6.1 is available everywhere?

Reply:
Good point, just checked and it's failing on Manjaro. There's no hurry to merge this anyway, so let's just wait until it's supported :)

Reply:
ping @SpotlightKid